src/libraries/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -2796,6 +2796,7 @@ public partial struct SqlBinary : System.Data.SqlTypes.INullable, System.ICompar
         void System.Xml.Serialization.IXmlSerializable.WriteXml(System.Xml.XmlWriter writer) { }
         public System.Data.SqlTypes.SqlGuid ToSqlGuid() { throw null; }
         public override string ToString() { throw null; }
+        public static SqlBinary WrapBytes(byte[] bytes) { throw null; }
```

Comment:
Have we agreed to drop this API?

https://github.com/dotnet/runtime/issues/51836#issuecomment-1190884318

Reply:
The one being referred to in that comment is the SqlGuid version which isn't needed if https://github.com/dotnet/runtime/pull/72549 is merged. 
This change is on SqlBinary and is still required and approved.